client/constants/ISSUE_CATEGORIES.json

```
@@ -191,24 +191,21 @@
     "Entitlement | Voided Enlistment"
   ],
   "vha": [
-    "Beneficiary Travel | Common Carrier",
-    "Beneficiary Travel | Mileage",
-    "Beneficiary Travel | Special Mode",
+    "Beneficiary Travel",
     "Caregiver | Eligibility",
     "Caregiver | Tier Level",
     "Caregiver | Revocation/Discharge",
     "Caregiver | Other",
-    "Clothing Allowance § 3.810(b) Certification",
-    "Eligibility for Fee Basis Care",
-    "Eligibility for Treatment | Dental",
-    "Eligibility for Treatment | Other",
-    "Family Member Services | CHAMPVA Eligibility",
-    "Family Member Services | CHAMPVA Medical Charges",
-    "Family Member Services | Camp Lejeune Family Member Eligibility",
-    "Family Member Services | Foreign Medical Program Medical Charges",
-    "Family Member Services | Spina Bifida Medical Charges",
-    "Level of Priority for Treatment",
+    "Clothing Allowance",
+    "Initial Eligibility and Enrollment in VHA Healthcare",
+    "Eligibility for Dental Treatment",
+    "Medical and Dental Care Reimbursement",
```

```suggestion
    "Clothing Allowance",
    "Medical and Dental Care Reimbursement",
    "Initial Eligibility and Enrollment in VHA Healthcare",
    "Eligibility for Dental Treatment",
```

This update will alter the order of the categories to match the way they are presented in the [designs](https://bipuiux.invisionapp.com/console/share/EAQMSNZ7FW2/975710611/play).